apps/frontend/src/libs/components/sidebar/sidebar-item/styles.module.css

```
@@ -0,0 +1,28 @@
+.navigation-link {
+	display: flex;
+	flex-direction: row;
+	gap: 8px;
+	align-items: center;
+	padding: 12px 16px;
+	color: var(--color-text-secondary);
+	text-decoration: none;
+}
+
+.navigation-link:hover {
+	color: var(--color-text-primary);
+	background-color: var(--color-background-secondary);
+	border-radius: 6px;
+}
+
+.navigation-link:hover svg {
+	color: var(--color-brand-primary);
+}
```

Wrap icon with a wrapper and add color property to it. see discussion in #51 
```suggestion
.navigation-link:hover .navigation-icon {
	color: var(--color-brand-primary);
}
```